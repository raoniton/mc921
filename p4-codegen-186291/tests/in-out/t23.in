class Program {
    String x = "test";

    public static void main(String[] args) {
        String y = "test";
        print("x: ", this.x);
        print();
        print("y: ", y);
        print();
        if (this.x == "test") {
            print("equal");
        }
        else {
            print("not equal");
        }
    }
}