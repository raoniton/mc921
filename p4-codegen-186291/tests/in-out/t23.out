x: test
y: test
equal